packages/server/graphql/mutations/pokerSetFinalScore.ts

```diff
@@ -137,7 +138,7 @@ const pokerSetFinalScore = {
             dimensionField?.fieldType === 'string' ? finalScore : Number(finalScore)
           await manager.updateStoryPoints(cloudId, issueKey, updatedStoryPoints, fieldId)
         } catch (e) {
-          return {error: {message: e.message}}
+          standardError(e)
```

Comment:
missing return statement?
for whatever reason, `Error` instances can get swallowed & unreported (maybe their stack trace is too long? :shrug: ), so I always just return a POJO.
We should return `{error: {message}}` here

Reply:
Interesting, that's good to know! It looks like [standardError](https://github.com/ParabolInc/parabol/blob/3ea2c761bdaa507c0319b6f0da83435b43ecfe51/packages/server/utils/standardError.ts) returns a POJO (when the return statement isn't missing 😅 ) and sends the error to Sentry. Still, I've pushed a change returning `{error: {message}}` as we weren't previously sending the error to Sentry